powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/v3/PowerAuthServiceImpl.java

```diff
@@ -480,6 +482,7 @@ public VaultUnlockResponse vaultUnlock(VaultUnlockRequest request) throws Generi
             final String applicationKey = request.getApplicationKey();
             final String signature = request.getSignature();
             final SignatureType signatureType = request.getSignatureType();
+            final String signatureVersion = request.getSignatureVersion();
```

Comment:
In the beginning of the `vaultUnlock` method in this class there is a request validation logic. Since the `signatureVersion` is a required parameter, let's add it into the validation logic.

See:
https://github.com/wultra/powerauth-server/blob/5b7914229ce22990abebc4a7cb008d1a3bb346c0/powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/service/v3/PowerAuthServiceImpl.java#L473